src/story/grid/mod.rs

```diff
@@ -7,6 +7,7 @@ use crate::story::common::position::Position;
 
 use self::elements_cache::ElementsCache;
 
+// TODO: I think this is a view concern...?
```

Comment:
I need a better home for this but IDK where just yet. I've got a NestRenderingPlugin but this logic is common to both Nest and Crater, so it probably shouldn't live there. I'll probably create a CommonRenderingPlugin here shortly.

Reply:
I'm redoing this for the `bevy_ecs_tilemap` replacement, so 100% fine to leave this as is.